docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,620 @@
+---
+title: "Using GCP Services"
```

Comment:
Can you change this to "GCP Service Guide" per @prasek.

Reply:
His comment suggested that actually https://github.com/crossplaneio/crossplane/pull/802/#pullrequestreview-289772546

Reply:
I'm pretty sure the thinking changed while you were asleep. See https://crossplane.slack.com/files/UJDEZ82R1/FN3B9Q8M8/image.png.